src/test/java/com/znsio/teswiz/steps/VodQASteps.java

```diff
@@ -37,4 +37,16 @@ public void scrollToElement() {
     public void isElementWithTextVisible() {
         new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).isElementWithTextVisible();
     }
+
+    @When("I scroll down by screen size on vertical swiping screen")
+    public void iScrollDownByScreenSizeOnVerticalSwipingScreen() {
+        new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).scrollDownByScreenSizeOnVerticalSwipingScreen();
+
+    }
+
+    @Then("Element lower in the list should be visible")
+    public void elementLowerInTheListShouldBeVisible() {
+        new VodqaBL(SAMPLE_TEST_CONTEXT.ME, Runner.getPlatform()).isElementWithTextVisible();
+
+    }
```

Comment:
No need for this, you can remove this. You can use already defined isElementWithTextVisible() as both does exactly the same thing. Refer to feature is needed

Reply:
removed duplicate method